### PR TITLE
Getting Started fixes

### PR DESCRIPTION
**Pull request by ale-kacperczyk**
- response format in **Authorization**
- removed the example from **Sign in with LiveChat**
- specified the token proprietor in **Messaging APIs in practice**

### CODE REVIEW

File: content/getting-started/authorization/index.mdx [hunk 1]
```diff
@@ -318,8 +322,12 @@ curl "https://accounts.livechatinc.com/token" \
 ```json
 {
   "access_token": "1/fFAGRNJru1FTz70BzhT3Zg",
+  "account_id": "c2ace1c1-8feb-4903-b7b2-05b8249a31c1",
+  "entity_id": "p.litwinowicz@livechatinc.com",
```
Comment (litwinow) [inline content/getting-started/authorization/index.mdx]: Please use documentation defaults from https://github.com/livechat/agent-api/#documentation-default-ids

File: content/getting-started/authorization/index.mdx [hunk 2]
```diff
@@ -368,8 +380,12 @@ curl "https://accounts.livechatinc.com/token" \
 ```json
 {
   "access_token": "1/fFAGRNJru1FTz70BzhT3Zg",
+  "account_id": "c2ace1c1-8feb-4903-b7b2-05b8249a31c1",
```
Comment (litwinow) [inline content/getting-started/authorization/index.mdx]: As above

File: content/getting-started/authorization/index.mdx (hunk 1 above)
Comment (ale-kacperczyk) [inline content/getting-started/authorization/index.mdx]:
```suggestion
  "entity_id": "agent1@example.com",
```

File: content/getting-started/authorization/index.mdx [hunk 3]
```diff
@@ -318,8 +322,12 @@ curl "https://accounts.livechatinc.com/token" \
...
```
Comment (ale-kacperczyk) [inline content/getting-started/authorization/index.mdx]:
```suggestion
  "account_id": "b7eff798-f8df-4364-8059-649c35c9ed0c",
```

File: content/getting-started/authorization/index.mdx (hunk 2 above)
Comment (ale-kacperczyk) [inline content/getting-started/authorization/index.mdx]:
```suggestion
  "account_id": "b7eff798-f8df-4364-8059-649c35c9ed0c",
```

File: content/getting-started/authorization/index.mdx [hunk 4]
```diff
@@ -368,8 +380,12 @@ curl "https://accounts.livechatinc.com/token" \
...
+  "entity_id": "p.litwinowicz@livechatinc.com",
+  "expires_in": 28800,
+  "license_id": 100000558,
+  "organization_id": "3808f8e6-7fe6-468c-a1bd-34e15e08c4eb",
```
Comment (ale-kacperczyk) [inline content/getting-started/authorization/index.mdx]:
```suggestion
  "organization_id": "390e44e6-f1e6-0368c-z6ddb-74g14508c2ex",
```

File: content/getting-started/authorization/index.mdx [hunk 5]
```diff
@@ -318,8 +322,12 @@ curl "https://accounts.livechatinc.com/token" \
...
```
Comment (ale-kacperczyk) [inline content/getting-started/authorization/index.mdx]:
```suggestion
  "organization_id": "390e44e6-f1e6-0368c-z6ddb-74g14508c2ex",
```

File: content/getting-started/authorization/index.mdx [hunk 6]
```diff
@@ -318,8 +322,12 @@ curl "https://accounts.livechatinc.com/token" \
 ```json
 {
   "access_token": "1/fFAGRNJru1FTz70BzhT3Zg",
+  "account_id": "c2ace1c1-8feb-4903-b7b2-05b8249a31c1",
+  "entity_id": "p.litwinowicz@livechatinc.com",
+  "expires_in": 28800,
+  "license_id": 100000558,
```
Comment (ale-kacperczyk) [inline content/getting-started/authorization/index.mdx]:
```suggestion
  "license_id": 104130623,
```

File: content/getting-started/authorization/index.mdx [hunk 7]
```diff
@@ -368,8 +380,12 @@ curl "https://accounts.livechatinc.com/token" \
...
```
Comment (ale-kacperczyk) [inline content/getting-started/authorization/index.mdx]:
```suggestion
  "license_id": 104130623,
```

File: content/getting-started/authorization/index.mdx [hunk 8]
```diff
@@ -368,8 +380,12 @@ curl "https://accounts.livechatinc.com/token" \
 ```json
 {
   "access_token": "1/fFAGRNJru1FTz70BzhT3Zg",
+  "account_id": "c2ace1c1-8feb-4903-b7b2-05b8249a31c1",
+  "entity_id": "p.litwinowicz@livechatinc.com",
```
Comment (ale-kacperczyk) [inline content/getting-started/authorization/index.mdx]:
```suggestion
  "entity_id": "agent1@example.com",
```

File: content/getting-started/authorization/index.mdx (hunk 1 above)
Comment (ale-kacperczyk) [reply content/getting-started/authorization/index.mdx]: Changed

File: content/getting-started/authorization/index.mdx (hunk 2 above)
Comment (ale-kacperczyk) [reply content/getting-started/authorization/index.mdx]: Changed

**Pull request merged**